example/12_reduce/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 add_example_executable(example_reduce_blockwise reduce_blockwise.cpp)
 add_example_executable(example_reduce_blockwise_two_call reduce_blockwise_two_call.cpp)
+add_example_executable(example_softmax_blockwise softmax_blockwise.cpp)
```

Comment:
I don't know whether it is a good choice to put software example under `example/12_reduce`

Reply:
right, softmax should be under its own example folder

Reply:
Addressed in https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/284/commits/c3e2a1ca5a13efe8836a98a4b2582beb539a0395